src/sql/base/browser/ui/chart/chart.component.ts

```diff
@@ -0,0 +1,154 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Component, Inject, forwardRef, ChangeDetectorRef } from '@angular/core';
+import * as chartjs from 'chart.js';
+import { mixin } from 'sql/base/common/objects';
+import { Disposable } from 'vs/base/common/lifecycle';
+import { BubbleChartPoint, ChartOptions, ScatterChartPoint } from 'azdata';
+
+@Component({
+	selector: 'chart-component',
+	templateUrl: decodeURI(require.toUrl('./chart.component.html'))
+})
+export class Chart<T extends ChartOptions> extends Disposable {
+
+	public _labels: string[];
```

Comment:
docstrings for all public properties, please.  I especially think having both `label` and `labels[]` is confusing.  How is each one used?